src-graphql/checkout-query.js

```diff
@@ -0,0 +1,11 @@
+import lineItemConnectionQuery from './line-item-connection-query';
+import shippingRateQuery from './shipping-rate-query';
+import mailingAddressQuery from './mailing-address-query';
+import baseQuery from './base-query';
+
+const defaultFields = ['id', 'ready', 'note', 'createdAt', 'updatedAt', 'requiresShipping', ['shippingLine', shippingRateQuery()],
```

Comment:
Can we format this in the long form?

```js
const defaultFields = [
  'id',
  'ready',
  // etc
];
```

Easier to read and modify in the future